contracts/test/forkMainnet/L2Deposit/Optimism.t.sol

```diff
@@ -24,9 +24,17 @@ contract TestL2DepositOptimism is TestL2Deposit {
         // overwrite deposit data with encoded optimism specific params
         DEFAULT_DEPOSIT.data = abi.encode(optL2Gas);
 
-        // compose payload with signature
-        bytes memory sig = _signDeposit(userPrivateKey, DEFAULT_DEPOSIT);
-        bytes memory payload = abi.encodeWithSelector(L2Deposit.deposit.selector, IL2Deposit.DepositParams(DEFAULT_DEPOSIT, sig));
+        // sign the deposit action
+        bytes memory depositActionSig = _signDeposit(userPrivateKey, DEFAULT_DEPOSIT);
+        // create spendWithPermit using the deposit and sign it
+        // _createSpenderPermitFromL2Deposit() will create EIP712 hash of the deposit for us
```

Comment:
This too

Reply:
Fixed!

Reply:
Missed this one?
But now that I look at comment being moved to `_createSpenderPermitFromL2Deposit`, I wasn't sure the comment helps much. What was the reason to add the comment in the first place?

Reply:
I think I added that comment to remind me the details when developing, I was a bit confused with EIP712 stuff at that time. But now I look at the code, I feel it's pretty self-explanatory. Let me remove it.